kuma/api/v1/views.py

```diff
@@ -268,7 +271,31 @@ def search(request, locale):
 
     # Return as many as 40 matches, since we're not implementing pagination yet
     response = search[0:40].execute()
-    return JsonResponse(response.to_dict())
+
+    hits = []
+    for hit in response:
+        # Be explicit about what to return.
+        # Only include things the React component might and will use.
+        hit_dict = hit.to_dict()
+        # Not every document has 'tags' so set this default.
+        hit_dict.setdefault('tags', [])
+        # 'locale' might get returned from hit.to_dict(). Remove it if there.
+        hit_dict.pop('locale', None)
+        # This one comes from the meta
+        hit_dict['score'] = hit.meta.score,
```

Comment:
I noticed that `hit.meta.score` seems to always be a list with a single item, the actual score, and that score is different than the score that's being used currently, `hit._score`. Does it make sense to assign the actual score to `hit_dict['score']` rather than the list, like `hit_dict['score'] = hit.meta.score[0]`? Or should we use `hit._score` instead?

Reply:
It's a list!? That's sad that I didn't even notice. Clearly, the score isn't being used.

Reply:
I'm feeling kinda aggressive today. Like with `tags`, if we're not displaying it, we shouldn't be sending it. APIs are hard to modify when they're established so it's always a good idea to work up slowly from the absolute minimal. 
So, the overall goal of this PR is to reduce the returned JSON to the absolute minimal that is used in the React UI view code.